projects/ardin_mb/ardin_mb.cc

```diff
@@ -50,25 +69,87 @@ int main(int argc, char *argv[])
         return EXIT_FAILURE;
     }
 
-    // Set clear colour to match matlab and enable depth test
-    glClearColor(0.0f, 1.0f, 1.0f, 1.0f);
+    // Tweak OpenGL settings
+    glClearColor(clearColour[0], clearColour[1], clearColour[2], clearColour[3]);
     glEnable(GL_DEPTH_TEST);
     glLineWidth(4.0);
     glPointSize(4.0);
 
     // Create memory
-    //Navigation::PerfectMemory<> memory(cv::Size(MBParams::inputWidth, MBParams::inputHeight));
+#ifdef NO_GENN
+    Navigation::PerfectMemory<> memory(cv::Size(MBParams::inputWidth, MBParams::inputHeight));
     //Navigation::InfoMax<float> memory(cv::Size(MBParams::inputWidth, MBParams::inputHeight), 0.01f);
+#else
     MBMemory memory;
+#endif
```

Comment:
This is a really minor nit, but I feel like ``NO_GENN`` probably isn't the best name for this option. It's not obvious that ``NO_GENN`` means perfect memory instead of MB model.

But it's not super important. I would probably stick some ``message()``s in the ``CMakeLists.txt`` file though saying which algo it's building for just to make it obvious.

Reply:
``MEMORY_TYPE`` in the style of robot type would perhaps be the best solution